### PR TITLE
Description for the objects slice

### PR DESCRIPTION
**Pull request by robertmin1**
Comment for the objects slice in p11mod.

### CODE REVIEW

File: p11mod/p11mod.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func init() {
 type llSession struct {
 	session         p11.Session
 	slotID          uint
-	objects         []p11.Object
+	objects         []p11.Object //Used to keep track of mapping between object handles and objects, because object handles don't exist in the p11 API
```
Comment (JeremyRand) [inline p11mod/p11mod.go:60]: Can you put this comment on the preceding line (as a standalone line)?  It's long enough that putting it on the same line as the code makes it a bit hard to read on small screens.

Comment (robertmin1) [reply p11mod/p11mod.go:60]: Okay!

**Comment by JeremyRand** [comment]
As per the `Go Lint Mandatory` warnings, can you add a space after the `//` and then run `gofmt -s` on the file?

**Comment by robertmin1** [comment]
Okay!

**Comment by JeremyRand** [comment]
Looks like `gofmt -s` is still needed.

**Comment by JeremyRand** [comment]
Can you squash down to 1 commit, and change commit message to imperative tense (e.g. maybe prepend "Add" to the commit message)?

**Comment by robertmin1** [comment]
Okay, no problem!

**Pull request closed without merging**